src/check_ast.rs

```diff
@@ -765,6 +765,13 @@ where
                     plugins::print_call(self, expr, func);
                 }
 
+                // flake8-comprehensions
+                if self.settings.enabled.contains(&CheckCode::C403) {
+                    if let Some(check) = checks::unnecessary_list_comprehension(expr, func, args) {
+                        self.checks.push(check);
+                    };
```

Comment:
I'm trying to establish a pattern whereby we add new checks to `plugins`, and every plugin takes `&mut self` as its first argument. But that might be overkill for these really simple cases. (Better when there's a lot of logic around the `checks::` call.)